### PR TITLE
Add the ability to use custom urls

### PR DESCRIPTION
**Pull request by dgorodnichy**

### CODE REVIEW

File: blog-urls.csv
Comment (pftg) [inline blog-urls.csv:1]: can we use `devto_urls_mapping.csv` or use `_` instead of `-`

File: bin/from_devto [hunk 1]
```diff
@@ -106,4 +152,4 @@ loop do
   page += 1
 end
 
-pp "TODO: Need to uncomment canonical url update when blog will be live."
+pp 'TODO: Need to uncomment canonical url update when blog will be live.'
```
Comment (pftg) [inline bin/from_devto]: `puts` better

**Review by pftg — CHANGES_REQUESTED** [review]
@dgorodnichy please add readme how to provide custom name for blog post from dev.to

File: bin/from_devto (hunk 1 above)
Comment (dgorodnichy) [reply bin/from_devto]: Done

File: blog-urls.csv
Comment (dgorodnichy) [reply blog-urls.csv:1]: Done

**Review by pftg — APPROVED** [review]

**Pull request merged**